test-tools/wamr-ide/VSCode-Extension/src/extension.ts

```diff
@@ -83,7 +88,9 @@ export async function activate(context: vscode.ExtensionContext) {
     typeMap.set('Debug', 'Debug');
     typeMap.set('Destroy', 'Destroy');
 
-    wasmTaskProvider = new WasmTaskProvider(typeMap, scriptMap);
+    wamrVersion = getWAMRExtensionVersion(context);
```

Comment:
could that be a constant declared here instead of making it `var` in L49?

Reply:
I guess it's perfectly fine to declare a const here, but I think maybe it's better just to follow the declare convention of other variables:)